README.md

```diff
@@ -111,13 +111,16 @@ Tested with the following software:
 
 Install dependencies (`sudo apt install libopencv-dev build-essential v4l2loopback-dkms curl`).
 
+Clone this repository with `git clone --recursive --depth=1 https://github.com/floe/deepbacksub.git`.
+
 Run `make` to build everything (should also clone and build Tensorflow Lite).
+Alternatively, you can also use `cmake`. Create a subfolder (e.g. `build`), change to that folder and run: `cmake -GNinja .. && ninja`.
```

Comment:
I know there are multiple generators for `cmake`, so using `ninja` is not required. Thus I'd refrain from explicitly suggesting it, because this looks like yet another dependency that a normal user might want to avoid. The default generator of cmake should be just fine, thus why not stick with it?

Reply:
It is slower for large things like tensorflow, but nevertheless, most people know about it thus they will automatically choose it. Changed README to Make generator.